yarn.lock

```diff
@@ -3129,6 +3129,13 @@ create-hmac@^1.1.0, create-hmac@^1.1.2, create-hmac@^1.1.4:
     safe-buffer "^5.0.1"
     sha.js "^2.4.8"
 
+cross-env@^7.0.3:
```

Comment:
all of this diff seems weird but I remember you told me yesterday that there were agora deps checked into source? I'm not seeing those removed here...I'm also not seeing any agora deps in the [`yarn.lock` in the develop branch](https://raw.githubusercontent.com/dequelabs/cauldron/develop/yarn.lock) either 🤔

Reply:
They were from my initial commit. I removed them by uninstalling and reinstalling the cross-env package

Reply:
i had my Deque npmrc pointed to agora when I checked in the first time.

Reply:
so this patch doesn't update any deps but has yarn.lock diffs?

Reply:
@schne324 - I reverted the `yarn.lock` changes that were caused by my DQ .npmrc config. I will need a re-thumb.